fakenet/listeners/POPListener.py

```diff
@@ -131,6 +134,12 @@ def handle(self):
 
                             handler = getattr(self, 'pop_%s' % (cmd.decode("utf-8").upper()), self.pop_DEFAULT)
```

Comment:
not an error from your side, but move cmd.decode("utf-8").upper() to the line above so that line 135 and 136 would look like:
```
cmd = cmd.decode("utf-8").upper()
handler = getattr(self, 'pop_%s' % (cmd), self.pop_DEFAULT)
```